falcon/response.py

```diff
@@ -434,3 +434,10 @@ def _wsgi_headers(self, media_type=None):
             return headers.items()
 
         return list(headers.items())  # pragma: no cover
+
+    def redirect(self, uri, permanent=False):
```

Comment:
Since this method isn't a property, I'd prefer calling it `set_redirect`. Also, could you move the method to before `def _wsgi_headers(...)`? I'd like to keep public properties and methods grouped together.